docs/topics/gradle/gradle-configure-project.md

```diff
@@ -179,9 +179,7 @@ Use the following code to set a toolchain. Replace the placeholder `<MAJOR_JDK_V
 
 ```kotlin
 kotlin {
-    jvmToolchain {
```

Comment:
I think we should show both ways to configure - with such shortcut or without it. Without it still could be useful for some cases.